wordCount.rb

```diff
@@ -0,0 +1,29 @@
+class WordCounter
+  attr_accessor :file, :input, :output
+  def initialize(file)
+    @file = file
+    @input = ""
+    @output =  Hash.new(0)
+  end
+
+  def parse
+    File.foreach(@file) {|x| count(x)}
+  end
+
+  def print
+    @output = @output.sort_by {|key, value| value}
+    @output.each {|key, value| puts "#{value} - #{key}"}
+  end
+
+  private
+  def count(input)
+    input = input.split
+    input.each do |word|
+      @output[word] += 1
+    end
+  end
+end
+
+program = WordCounter.new("speech.txt")
+program.parse
+program.print
```

Comment:
All wrapped up in a class, nice! This looks pretty good, just have a couple small suggestions.

You're referencing your instance variables by their @names all over your methods, but since you've declared attr_accessor for them all at the top of your class, you can drop the @ inside of your methods (except initialize). The difference that this makes is that instead of directly accessing your instance variables, you would be accessing them through the getter method created by attr_accessor. Functionally this is the same thing for this class! In other cases, you might want to define a custom attribute getter for a class and rely on that instead of directly accessing a variable.

Also, since you're not externally setting any of `@file`, `@input`, or `@output`, you might consider using attr_reader instead of attr_accessor, since you don't need the writer methods.